.ci_support/linux_64_cuda_compiler_version11.0.yaml

```diff
@@ -42,6 +42,11 @@ librdkafka:
 - '1.6'
 ncurses:
 - '6.2'
+numpy:
+- '1.18'
+- '1.18'
+- '1.18'
```

Comment:
I wonder why do you need three equal entries?..

Reply:
I guess these three entries correspond to python versions 3.6, 3.7, and 3.8, and the subsequent entry 1.19 corresponds to python version 3.9.